tests/unit/cli/config_test.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from tests import unittest
+
+from fig.cli import config
+from fig.cli.errors import UserError
+
+import yaml
+import os
+
+
+TEST_YML_DICT_WITH_DEFAULTS = yaml.load("""db:
+  image: postgres
+web:
+  build: ${DJANGO_BUILD:.}
+  command: python manage.py runserver 0.0.0.0:8000
+  volumes:
+    - .:/code
+  environment:
+    DJANGO_ENV: ${TEST_VALUE:production}
+  ports:
+    - ${DJANGO_PORT:"8000:8000"}
```

Comment:
Interested to see a test case and/or instructions for providing port numbers <= 60 via environment variables, to test for this: https://github.com/docker/fig/blob/master/docs/yml.md#ports. Don't know if it's possible to do, but thinking of:

    - ${PORT_A}:${PORT_B}

How should that be done to prevent fig from interpreting them incorrectly?